python/ray/rllib/policy/tf_policy.py

```diff
@@ -421,9 +421,9 @@ def _build_compute_actions(self,
         builder.add_feed_dict({self._obs_input: obs_batch})
         if state_batches:
             builder.add_feed_dict({self._seq_lens: np.ones(len(obs_batch))})
-        if self._prev_action_input is not None and prev_action_batch:
+        if self._prev_action_input is not None and prev_action_batch is not None:
```

Comment:
what does this change do?

Reply:
This is related to creating a new local policy evaluator called old_policy_worker. The line above errors out if a new local worker is instantiated. Adding "is not None" ensures the same functionality of the code as before and resolves instantiation issues.